amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/parser/spec/domain/OasResponseParser.scala

```diff
@@ -100,10 +100,12 @@ case class OasResponseParser(map: YMap, adopted: Response => Unit)(implicit ctx:
 
         // OAS 3.0.0
         if (ctx.syntax == Oas3Syntax) {
-          map.key(
-            "content",
-            entry => payloads ++= OasContentsParser(entry, res.withPayload).parse()
-          )
+          map
+            .key("content")
+            .map(entry => {
+              res.set(ResponseModel.Payloads, AmfArray(payloads, Annotations(entry)))
```

Comment:
This won't work as it will be overriden in line 123. I would patch it in the if in line 122 if you want to finish it fast. You can also refactor this shit class if you like also to run the change nicely.

Reply:
Could be overwritten but not necessarily. The idea behind the patch is to handle the case where we have the key "content" but the value is empty, resulting in an empty payload(*) list. The spec would still be valid, an AMF provided no real way to know if the key was present or not. 

* Except for the case where the RAML response payload had been provided.